app/templates/model-details.html

```diff
@@ -239,22 +239,22 @@ <h2 class="panel-title">{{activeModel.data.name}}</h2>
             </p>
 
             <div class="form-group input-group" @click.stop.prevent="toggle('export_d3dinput', true)">
-              <span class="input-group-addon"><input type="checkbox" class="downloadoption" value="export_d3dinput" v-model="selectedDownloads.export_d3dinput" @click.stop="return false;"></span>
+              <span class="input-group-addon"><input type="checkbox" class="downloadoption" value="export_d3dinput" v-model="selectedDownloads.export_d3dinput" @click.stop="doNothing"></span>
```

Comment:
Perhaps include 
`<!-- @click.stop="doNothing" captures an event and avoids bubbling it down, so we don't get 2 toggles -->`